pkgs/applications/version-management/mercurial/default.nix

```diff
@@ -50,7 +50,8 @@ in python2Packages.buildPythonApplication {
       chmod u+x $out/share/cgi-bin/hgweb.cgi
 
       # install bash completion
-      install -D -v contrib/bash_completion $out/share/bash-completion/completions/mercurial
+      install -v -m644 -D contrib/bash_completion $out/share/bash-completion/completions/mercurial
```

Comment:
Completion doesn't work in bash it seems, we need to copy to [`hg`,  `_hg` or `hg.bash`](https://github.com/scop/bash-completion/blob/master/bash_completion#L2052) for `bash-completions` to lazy load the file. Should probably fix it while we're here :)